README.md

```diff
@@ -18,7 +18,7 @@ If you are wondering where all the recipes went in v7.0+, or how on earth I use
 - Red Hat/CentOS/Scientific 6+
 - Fedora
 
-## PostgreSQL version
+### Postgresql version
```

Comment:
When it isn't in all caps the PostgreSQL notes it with SQL capitalized
https://www.postgresql.org/about/